README_js.md

```diff
@@ -353,6 +353,17 @@ import { v4 as uuidv4 } from 'uuid';
 Workers](https://caniuse.com/#feat=cryptography) and we are not aware of a polyfill (let us know if
 you find one, please).
 
+## Developing
```

Comment:
This isn't of interest to most readers.  Move to [PR template](https://help.github.com/en/github/building-a-strong-community/creating-a-pull-request-template-for-your-repository) (preferably the hidden `.github/pull_request_template.md` file).